wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+| [Approach rate](/wiki/Beatmap/Approach_rate) (AR) | Regulates the speed hit objects appear in. | ![][osu!] ![][osu!catch] |
+| [Accuracy](/wiki/Gameplay/Accuracy) | Measures how good the player hits hit objects on time. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| Key count | Specifies the number of keys used when playing a beatmap. | ![][osu!mania] |
+
+At the bottom of the statistics panel is the [star rating](/wiki/Beatmap/Star_rating) detailing how much skill the player requires for the beatmap.
+
+::: Infobox
+![](img/user-rating.png "User rating")
+:::
+
+If the beatmap is [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked), or [loved](/wiki/Beatmap/Category#loved), a user rating will be displayed below the statistics panel. User rating is a feature in osu!(stable) where after passing a beatmap users are able to vote between 1 and 10 stars based on how much they liked it.
+
+If the user votes 6 stars or more (positive vote), the bar will be coloured green, otherwise yellow (negative vote). The numbers to the side of the bar describe the number of users that have voted either positively or negatively. Above the bar is a number in brackets displaying the average number of stars every user voted for.
+
+The rating spread below the user rating illustrates the proportionate amount of user votes for each number of stars.
+
+## Metadata
+
+### Beatmap description
+
+The beatmap description is a custom field often used for:
+
+- linking ressources such as the background image or hitsound samples
+- commending other users for their help (e.g. guest mappers, modder, storyboarder)
+- trivia facts related to the beatmap (e.g. mapping milestones)
+
+### Keywords
+
+In addition to gameplay-related statistics, every beatmap includes metadata fields to help categorise the beatmap's song. Before submitting a beatmap as part of the [ranking process](/wiki/Beatmap_ranking_procedure), a mapper is required to add this information to the beatmap.
+
+- [Genre](/wiki/Beatmap/Genre_and_language#list-of-genres): The musical or media genre this song mainly uses.
+- [Language](/wiki/Beatmap/Genre_and_language#list-of-languages): In which language the lyrics are, or instrumental if no lyrics are present.
+- [Mapper tags](/wiki/Beatmap/Beatmap_tags#mapper-tags): Useful keywords containing background information about the song.
+- **Source:** The original media the song was produced for, such as a video game or movie.
+
+The nominator section is added as soon as the beatmap is nominated by [beatmap nominators](/wiki/People/Beatmap_Nominators) for becoming ranked.
+
+After the beatmap is ranked, users are able to vote for [user tags](/wiki/Beatmap/Beatmap_tags#user-tags). Upon reaching 5 votes on an user tag, the respective tag will be displayed in the `user tag` segment in this section.
```

Comment:
Would want to mention this is a lazer exclusive feature.
```suggestion
After the beatmap is ranked, users are able to vote for [user tags](/wiki/Beatmap/Beatmap_tags#user-tags) from the osu!(lazer) client. Upon reaching 5 votes on an user tag, it will be displayed in the `user tag` segment in this section.
```